effects/nixops/run-nixops.nix

```
@@ -73,9 +73,25 @@ args@{
   # Defaults to pkgs.path
   NIX_PATH ? "nixpkgs=${path}",
 
+  # specify an action for the deploy which are mutually exclusive,
+  # options: activate, evaluate, plan, build, create, copy, dry-activiate, test, boot
```

```suggestion
  # options: switch, evaluate, plan, build, create, copy, dry-activate, test, boot
```

I was wrong earlier; should've said `switch` instead of `activate`.